### DISCUSSION TITLE
Pulling Image leads to: layers from manifest don't match image configuration

### DISCUSSION THREAD

**Discussion by pwfraley** [open-webui/open-webui#2636] [General]
# Bug Report

## Description
Trying to pull the image causes an error: layers from manifest don't match image configuration

**Bug Summary:**
[Provide a brief but clear summary of the bug]

**Steps to Reproduce:**
Docker version 26.1.3, build b72abbb

```composefile
    image: ghcr.io/open-webui/open-webui:main
    hostname: open-webui
    container_name: open-webui
    restart: unless-stopped
    ports:
      - 8080:8080
    environment:
      - 'OLLAMA_BASE_URL=http://ollama:11434'
      - 'WEBUI_SECRET_KEY=supersecret'

```

docker compose pull

**Expected Behavior:**
Image is pulled and usable.

**Actual Behavior:**
layers from manifest don't match image configuration

## Environment

- **Open WebUI Version:** see docker compose file
- **Ollama (if applicable):** n/a

- **Operating System:** Fedora 40, with official docker repos enabled and installed
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by justinh-rahb** [comment]
Another Fedora 40 user reported the same issue.

**Comment by pwfraley** [comment]
Yes, this is on Fedora 40

**Comment by hatsat32** [comment]
I have the same issue. Any possible solution?

**Comment by backd00rbenny** [comment]
I too am using Fedora 40 and received the same error. I'm seeing a trend here...
I used the install instructions from https://docs.openwebui.com/ and the Docker instructions here https://docs.docker.com/engine/install/fedora/

**Comment by jzerohn** [comment]
Can confirm the issue on a freshly installed Fedora 40.

However, whatever the issue is, pulling the newest container build works for me (using [open-webui:git-fec5134](https://github.com/open-webui/open-webui/pkgs/container/open-webui/222946594?tag=git-fec5134) while I'm typing this). So the issue hopefully is already fixed for the upcoming main version.

**Comment by RajSolai** [comment]
Is this only a fedora issue ?

**Reply by pwfraley** [reply]
On my Ubuntu 24.04 it runs fine, so seems to be Fedora 40 only problem

**Comment by andersonjwan** [comment]
Yes, I too have a fresh installed Fedora 40; and it seems to be a common problem. I was trying to pull the `carlasim/carla:0.9.15`. However, any tagged image from this repository doesn't seem to work.

**Comment by jabeston** [comment]
Issue still persists in Fedora/Nobara 40

**Comment by Wiggoggs** [comment]
Can confirm the same issue is present on my bazzite install

**Comment by tom-gora** [comment]
Same on Fedora server 40

**Comment by rmcmilli** [comment]
I was able to work around it by renaming unpigz based on this comment: 
 https://github.com/ollama/ollama/issues/5979#issuecomment-2263009015_

**Comment by JohnMastodon** [comment]
based on rmcmilli comment i fixed it on a a fedora immutable install (bazzite) with
```
rpm-ostree override remove pigz
docker pull ollama/ollama:rocm
rpm-ostree override reset pigz
```

pigz doesn't seem to be a dependency of anything but I prefer not to keep overriden defaults, hoping the next built ollama image won't have this issue
